src/track/beats.h

```diff
@@ -258,11 +254,20 @@ class Beats final : public QObject {
     /// Prints debugging information in stderr
     friend QDebug operator<<(QDebug dbg, const BeatsPointer& arg);
 
-  private slots:
-    void slotTrackBeatsUpdated();
+    /**
+     * Set sample rate of the track which owns these beats.
+     * @param sampleRate Track sample rate
+     */
```

Comment:
Use `///` for documentation comments

Reply:
I could not find any way to annotate function parameters with '///' comments. I think it is important to describe the parameters. For starters, I can see a nice and detailed function definition when I hover at a function in my IDE.